charts/bookkeeper/README.md

```diff
@@ -51,9 +51,9 @@ The following table lists the configurable parameters of the Bookkeeper chart an
 | `resources.limits.cpu` | Limits for CPU resources | `2000m` |
 | `resources.limits.memory` | Limits for memory resources | `4Gi` |
 | `storage.ledger.className` | Storage class name for bookkeeper ledgers | `standard` |
-| `storage.ledger.volumeSize` | Requested size for bookkeeper ledger persistent volumes | `10Gi` |
+| `storage.ledger.volumeSize` | Requested size for bookkeeper ledger persistent volumes | `100Gi` |
```

Comment:
I don't know if we should change that. Users wanting to deploy a small cluster for test do not need 100GB of ledger/journal volumes.